version/gallerydb.py

```diff
@@ -309,7 +324,7 @@ def modify_gallery(series_id, title=None, profile=None, artist=None, info=None,
 		if type:
 			assert isinstance(type, str)
 			executing.append(["UPDATE series SET type=? WHERE series_id=?", (type, series_id)])
-		if fav:
+		if fav != "":
```

Comment:
Why this change?
`fav` is an `int` (either `0` or `1`) if passed or `None` (the default) so this condition doesn't make sense (because it will always evaluate to `True`).
Or have i missed something? :)

Just realized that the previous condition would evaluate to `False` if `fav` was `0`! So you were almost right.
This should work:
```
if fav != None:
    # code
```

Might be a good idea to change them all to this to avoid any bugs in the future.